sandbox/settings.py

```diff
@@ -257,8 +257,8 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         # e.g: for user.username = johndoe, /u/(username) will be /u/johndoe
         "PROFILE_URLS": values.ListValue(
             [
-                {"label": _("Profile"), "href": _("{base_url:s}/u/(username)")},
-                {"label": _("Account"), "href": _("{base_url:s}/account/settings")},
```

Comment:
I don't think it's useless to translate path!?

Reply:
In case of url rewriting ?

Reply:
useless for sandbox ok, but not in other sites...